src/parser/shaman/enhancement/modules/core/FlametongueRefresh.tsx

```diff
@@ -1,33 +1,36 @@
 import React from 'react';
+import { Trans } from '@lingui/macro';
+
 import SPELLS from 'common/SPELLS';
 import { formatPercentage } from 'common/format';
-
 import Analyzer, { SELECTED_PLAYER } from 'parser/core/Analyzer';
 import Events, { ApplyBuffEvent, CastEvent, RefreshBuffEvent } from 'parser/core/Events';
-import { Trans } from '@lingui/macro';
 
-const PANDEMIC_THRESHOLD = 11500;//don't refresh with more than 4.5 seconds
-                                 // left on Flametongue buff
+// Don't refresh with more than 4.5 seconds left on Flametongue buff
+const PANDEMIC_THRESHOLD = 11500;
 
 class FlametongueRefresh extends Analyzer {
-  protected flametongueTimestamp = 0;
-  protected flametongueCasts = 0;
-  protected earlyRefresh = 0;
+  protected flametongueTimestamp: number = 0;
+  protected flametongueCasts: number = 0;
+  protected earlyRefresh: number = 0;
 
   constructor(options: any) {
     super(options);
-    this.active
-      = !this.selectedCombatant.hasTalent(SPELLS.SEARING_ASSAULT_TALENT.id);
+
+    this.active = !this.selectedCombatant.hasTalent(SPELLS.SEARING_ASSAULT_TALENT.id);
```

Comment:
Is it intended to mark this as active if they DONT have Searing Assault?

Reply:
For the current support this is correct. Usually what you do with Flametongue is refresh your own buff, which has a 16 second duration. However, this talent will force you to also refresh a debuff on your target which lasts for 6 seconds. As you can see this will conflict with each other.